Scripts/Player/Player.gd

```diff
@@ -5,6 +5,8 @@ var ZoomSpeed = 7
 var MaxZoom = 1.6
 var MinZoom = .5
 @onready var Highlight = $Sprite2D
+@onready var game_menu = $"../GameMenu"
```

Comment:
For the game menu. I think it'd be a bit better to have a helper function to get it.

Just in case the relative paths change

I am okay with paths going down the tree, but not back up the tree~